static/js/taxbrain.js

```diff
@@ -143,8 +146,13 @@ $('#current-year-modal').on('hide.bs.modal', function (e) {
   $('#start-year-select option[value="' + currentYear + '"]').attr("selected", "selected");
 });
 
-var Data = $('#cps-or-puf').val();
-$('#cps-or-puf').change(function(e) {
-    $('#cps-or-puf option').removeAttr("selected");
-    $('#cps-or-puf option[value="' + Data + '"]').attr("selected", "selected");
+var dataSource = $('#data-source-select').val();
+$('#data-source-select').change(function(e) {
+    $('#data-source-link').attr('href', '/taxbrain/?data_source=' + $(this).val());
+    $('#data-source-modal').modal('show');
+});
+
+$('#data-choice-modal').on('hide.bs.modal', function (e) {
+  $('#data-source option').removeAttr("selected");
+  $('#data-source option[value="' + dataSource + '"]').attr("selected", "selected");
```

Comment:
I changed this part in an effort to be exactly the same as the `start_year` javascript code. However, I'm not sure if it is working exactly right.